.github/workflows/ci.yml

```diff
@@ -11,18 +11,15 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  xcode-build-12:
-    name: Xcode 12 Build
-    runs-on: macOS-11
```

Comment:
well I broke Xcode 12 in addition to Xcode 11 builds in #308. Not ideal, but for the same reason we let #308 through, we're okay letting this change through.

Latest supported Xcode is 14. It'll be 15 on Friday.